ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-tests.robot

```diff
@@ -0,0 +1,96 @@
+*** Settings ***
+Documentation     Codeflare operator E2E tests - https://github.com/opendatahub-io/codeflare-operator/tree/main/test/odh
+Suite Setup       Prepare Codeflare E2E Test Suite
+Suite Teardown    Teardown Codeflare E2E Test Suite
+Library           Process
+
+
+*** Variables ***
+${CODEFLARE_DIR}            codeflare-operator
+${CODEFLARE_REPO_URL}       %{CODEFLARE_REPO_URL=https://github.com/opendatahub-io/codeflare-operator.git}
+${CODEFLARE_REPO_BRANCH}    %{CODEFLARE_REPO_BRANCH=main}
+
+
+*** Test Cases ***
+Run TestInstascaleMachinePool test
+    [Documentation]    Run Go E2E test: TestInstascaleMachinePool
+    [Tags]  Sanity
+    ...     Tier1
```

Comment:
InstaScale test is a long and disruptive one, I think it should be moved to Tier 2 or 3 (not sure which one is dedicated for such tests).

Reply:
Instascale tests were removed from this PR. Don't need to use Tier3 anymore here.